web/frontend/src/components/slam-2d-render.vue

```diff
@@ -161,16 +262,30 @@ const updateCloud = (pointcloud: Uint8Array) => {
     }
   }
 
-  scene.add(points);
-  scene.add(marker);
-  scene.add(intersectionPlane);
-};
+  // construct grids
+  const axesHelper1 = createAxisHelper('AxesPos', Math.PI / 2)
+  const axesHelper2 = createAxisHelper('AxesNeg', -Math.PI / 2)
+
+  // this needs to be updated so it is set to 1m
+  // have these be constants at the top
+  const gridHelper = new THREE.GridHelper(1000, 100, backgroundGridColor, backgroundGridColor);
```

Comment:
[nit] consider having a createGridHelp function like the axes for simplicity/consistency (but resolve micheals comment first!)